src/index.html

```diff
@@ -3,107 +3,181 @@
-
-      </div> <!-- /.wrapper -->
-    </div>
-
-    <div class="col-4 gray-box">
-      <!-- STAY TUNED -->
-      <header class="tabbed-header tabbed-pad-top">
-        <h3 class="subhead tabbed-heading">Stay tuned</h3>
-      </header>
-      <p>We’re already working on more tools that make it easier to shop for a mortgage and help you understand the mortgage process. Sign up for our email list, and we’ll let you know when we add more resources.</p>
-
-      <form class="signup2 inline-signup" id="signup" action="https://public.govdelivery.com/service/process_ss.xml" method="GET" data-thanks="Thanks, we’ll be in touch!">
-        <input type="hidden" name="code" value="USCFPB_46">
-        <label for="email" class="visuallyhidden">Email address</label>
-        <input type="email" id="email" name="email" placeholder="Enter your email address">
-        <fieldset class="submit_btn"><p><button id="beta-btn" class="btn">Sign up</button></p></fieldset>
-      </form>
-
-      <!-- RELATED LINKS -->
-      <header class="tabbed-header tabbed-pad-top">
-        <h3 class="subhead tabbed-heading">Related links</h3>
-      </header>
-
-      <ul class="links-ul">
-        <li>Looking for home buying help or struggling to repay your mortgage?<br />
-          <a class="link-with-icon" href="/find-a-housing-counselor/" target="_blank">Find a HUD-approved housing counselor<span class="cf-icon cf-icon-right"></span></a></li>
-        <li>Have more mortgage questions?<br />
-          <a class="link-with-icon" href="/askcfpb/search/?selected_facets=category_exact:mortgages" target="_blank">Get answers to common questions<span class="cf-icon cf-icon-right"></span></a></li>
-        <li>Have a complaint about your mortgage? <br />
-          <a class="link-with-icon" href="/complaint/#mortgage" target="_blank">Submit a mortgage complaint<span class="cf-icon cf-icon-right"></span></a></li>
-        <li>Wondering about our other mortgage work? <br />
-          <a class="link-with-icon" href="/mortgage/" target="_blank">Learn how CFPB is protecting mortgage borrowers<span class="cf-icon cf-icon-right"></span></a></li>
-      </ul>
+{% block content %}    
+<main class="content content__2-1 content__bleedbar" id="main" role="main">
+
+    <section class="content_hero">
+      
+      <!-- BACKGROUND IMAGE FOR THE HERO -->
+      <div id="hero-bg-image"></div>
+      
+       <!-- HERO -->
+      <div class="content_wrapper content_no-sidebar">
+          <section class="content_main">
+            <h1>Owning a Home</h1>
+            <h2>We're here to help you with your home-buying process. Learn what to expect and what questions to ask, and get tools and resources to help you make decisions.</h2>
+          </section>
+      </div>
+    </section>
+
+    <div class="content_wrapper">
+        <section class="content_main">
+
+          <section class="section__intro">      
+            <div class="content-l_col content-l_col-1-2">
```

Comment:
Oh, yeah, it looks like `.content-l__main` stacks `.content-l_col-1-2` at 801-899px. We could use `.content-l` without the modifier to avoid inheriting those breakpoints (or use the `.content-l__full` modifier instead -- it only stacks `col-1-2` on mobile).